1/src/main.rs

```diff
@@ -38,15 +39,14 @@ impl Response {
     /// and returns it in its byte encoding
     fn to_bytes(&self) -> Vec<u8> {
         let msg = match *self {
-            Response::Hello(ref id)        => format!("HELLO {}", id),
+            Response::Hello(ref id) => format!("HELLO {}", id),
             Response::Solution(ref answer) => format!("{}", answer),
-            Response::Bye(ref secret)      => format!("BYE {}", secret),
+            Response::Bye(ref secret) => format!("BYE {}", secret),
         };
```

Comment:
Because it makes changing stuff annoying since you have to reline it up. I'm generally opposed to adding whitespace within lines.